src/main/java/org/aksw/simba/lemming/ColouredGraph.java

```diff
@@ -173,10 +173,12 @@ public ColourPalette getDataTypedEdgePalette() {
         return dtEdgePalette;
     }
 
+    //Getter of NeighborhoodsArray are used only in NMSimilarity, but that similarity is deprecated!!!
+    @Deprecated
     public int[][] getInNeighborhoodsArray() {
         return graph.getInNeighborhoods();
     }
-
+    @Deprecated
```

Comment:
In Lemming the both getters only used by NMSimilarity. I think, if NMSimilarity is deprecated, then the both getters should also be deprecated??

Reply:
Little reminder : The complete version of this branch is in our forked repository. 
https://github.com/UPB-Lemming-Group/Lemming/pull/5

Reply:
I don't think so. Just because a functionality that has been used in some older classes is not needed at the moment, it might be still useful in the future. As long as these two getters do not create issues, I would argue to keep them.

Reply:
That means, we can free to make PR into dice-group repository??

Reply:
I mean that I won't review https://github.com/UPB-Lemming-Group/Lemming/pull/5 because it is a pull request from `UPB-Lemming-Group:improve_metrics_01` into `UPB-Lemming-Group:GeologyDataset`. So it simply merges changes from one of your branches into another branch of your project.

This PR here merges changes from the project group's project `UPB-Lemming-Group:improved_Metrics_sub01` into `dice-group:pgmerge`. So it will change the content of the dice project and I have to review these changes :wink: